packages/design-system-react/package.json

```diff
@@ -40,7 +40,7 @@
     "changelog:validate": "../../scripts/validate-changelog.sh @metamask/design-system-react",
     "clean:icons": "rimraf src/components/Icon/icons",
     "create-component": "ts-node scripts/create-component",
-    "generate-icons": "yarn clean:icons && svgr --config-file src/components/Icon/.svgrrc.js -d src/components/Icon/icons src/components/Icon/assets/*.svg && yarn generate-icons:index && yarn workspace @metamask/metamask-design-system prettier --write \"packages/design-system-react/src/components/Icon/icons/**/*.{ts,tsx}\"",
+    "generate-icons": "yarn clean:icons && svgr --config-file src/components/Icon/.svgrrc.js -d src/components/Icon/icons src/components/Icon/assets/*.svg && yarn generate-icons:index",
```

Comment:
Removed the Prettier step for generated icon files because these files are automatically generated by the script. Formatting them with Prettier is unnecessary.